packages/cli/src/ui/components/Footer.tsx

```diff
@@ -147,24 +146,26 @@ export const Footer: React.FC = () => {
               <ContextUsageDisplay
                 promptTokenCount={promptTokenCount}
                 model={model}
-                terminalWidth={terminalWidth}
+                terminalWidth={mainAreaWidth}
               />
             </Text>
             {showMemoryUsage && <MemoryUsageDisplay />}
           </Box>
-          <Box alignItems="center" paddingLeft={2}>
+          <Box alignItems="center">
```

Comment:
this paddingLeft={2} was causing the footer to be less wide than the InputPrompt so fixed instead adding padding only as needed to make sure things were separated by a space.